pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -417,13 +419,13 @@ func (service *SecurityPolicyService) buildAppliedGroupName(obj *v1alpha1.Securi
 	var rule *v1alpha1.SecurityPolicyRule
 	if ruleIdx != -1 {
 		rule = &(obj.Spec.Rules[ruleIdx])
-		ruleName := fmt.Sprintf("%s-%d", obj.Name, ruleIdx)
+		ruleName := fmt.Sprintf("%s_%d", obj.Name, ruleIdx)
```

Comment:
Do you plan use another change to remove ruleIdx from the NSX rule id/name ?

Reply:
Yes, I'm planning to do it once this PR is merged. So, we can narrow the scope of this PR.